src/locales/en_US.js

```diff
@@ -688,6 +688,10 @@ const strings = {
   update_now: 'Update Now',
   update_later: "I'll do it later",
 
+  error_boundary_title: 'Oops',
+  error_boundary_message:
+    'We are sorry, but something has gone wrong. Please report this crash to support@edge.app, and we will try to fix the bug.\n\nFeel free to kill and restart the app.',
```

Comment:
Please change text to 

"We're sorry but something went wrong. Please kill and restart the app to continue using Edge.\n\nIf the problem persists, contact us at support@edge.app, and we'll do our best to fix the problem."